### PR TITLE
refactor Spotlight bootstrapping

### PR DESCRIPTION
**Pull request by fdore**
- move serviceStub to context
- launch app on click, instead of on enter, so we can potentially support multiple results

### CODE REVIEW

File: src/client/src/apps/SpotlightRoute/handleIntent.ts [hunk 1]
```diff
@@ -29,10 +29,10 @@ function showCurrencyPair(currencyPair: string, { window }: PlatformAdapter) {
 }
 
 export const handleIntent = (
-  response: DetectIntentResponse[],
+  response: DetectIntentResponse,
   platformAdapter: PlatformAdapter,
 ) => {
-  const queryResult = response[0].queryResult
+  const queryResult = response.queryResult
```
Comment (sizov) [inline src/client/src/apps/SpotlightRoute/handleIntent.ts]: why this has changed?

Comment (sizov) [reply src/client/src/apps/SpotlightRoute/handleIntent.ts]: is it not an array anymore?

Comment (sizov) [reply src/client/src/apps/SpotlightRoute/handleIntent.ts]: i see now, ignore please

File: src/client/src/apps/SpotlightRoute/Spotlight.tsx [hunk 2]
```diff
@@ -109,11 +109,13 @@ export const Spotlight: FC = () => {
       <Response>
         {contacting ? (
           <>
-            <AdaptiveLoader size={14} speed={0.8} seperation={1.5} type="secondary" />
+            <AdaptiveLoader size={14} speed={0.8} seperation={1.5} type="secondary"/>
             <Contacting>Contacting…</Contacting>
           </>
         ) : (
-          response
+          <Suggestion onClick={() => handleIntent(response, platform)}>
```
Comment (sizov) [inline src/client/src/apps/SpotlightRoute/Spotlight.tsx]: as i understand you want user to explicitly select an intent, is this correct? wouldn't it be more "responcive" if we handle intent straightway? (at least when we know there's a single intent)

File: src/client/src/apps/SpotlightRoute/handleIntent.ts (hunk 1 above)
Comment (fdore) [reply src/client/src/apps/SpotlightRoute/handleIntent.ts]: I guess I can leave it as it was. I'm just not sure we ever get more than one result

**Pull request merged**